src/lib/chatSocket.ts

```diff
@@ -24,7 +24,7 @@ export default function useChatUtils(socket: Socket | undefined) {
               return;
             }
 
-            resolve(sentMessage);
+            resolve(message);
```

Comment:
why the change here?

Reply:
resolve(sentMessage) returns `{status: 'value'}` which causes another room not found error. we need to return message info
![Screenshot (216)](https://github.com/Dun-sin/Whisper/assets/97130553/a1fd88e5-762a-4d0e-9f38-a59267655fb9)

Reply:
can i see the console, try consoling the sentMessage in the chatSocket, cuz it's meant to return the message from the server

Reply:
you're right. I just checked and it seems like last time i was getting a return value with `status: value` because SendMessage handler didn't get triggered. I'll change this

Reply:
alright then